cdap-app-templates/cdap-etl/cdap-etl-core/src/main/java/co/cask/cdap/etl/common/DefaultMacroEvaluator.java

```diff
@@ -50,8 +57,13 @@ public String lookup(String property) {
 
   @Override
   @Nullable
-  public String evaluate(String macroFunction, String... arguments) {
-    // todo: use secure store macro function or logicalStartTime macro function.
-    return null;
+  public String evaluate(String macroFunction, String... arguments) throws InvalidMacroException {
+    if (macroFunction.equals(LOGICAL_START_TIME_FUNCTION_NAME)) {
+      return logicalStartTimeMacro.evaluate(logicalStartTime, arguments[0]);
```

Comment:
should pass all the arguments instead of just the first, and remove the parsing logic in LogicalStartTimeMacro